.buildkite/release-automation/verify-macos-wheels.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -euo pipefail
+
+set -x
+
+python_versions=("3.8" "3.9" "3.10" "3.11")
+
+# Check arguments
+if [ "$#" -ne 1 ]; then
```

Comment:
nit: use double bracket

also, you do not have to double quote `$#` double quotes are for strings. numbers do not need double quoting (though double quoting is also fine)

`if [[ $# -ne  1 ]]; then`

more details:

https://google.github.io/styleguide/shellguide.html#quoting
https://google.github.io/styleguide/shellguide.html#test----and---

probably just read the full styling guide :)

Reply:
fixed!